### PR TITLE
add hybrid cloud page

### PR DESCRIPTION
**Pull request by nastyapash**

### CODE REVIEW

File: qdrant-landing/themes/qdrant-2024/assets/css/partials/_hybrid-cloud-cases.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      color: $neutral-30;
+    }
+  }
+
+  @include media-breakpoint-up(xl) {
+    padding-top: $spacer * 2.5;
+    padding-bottom: $spacer * 6.5;
+
+    &__wrapper {
+      max-width: rem(1134);
```
Comment (trean) [inline qdrant-landing/themes/qdrant-2024/assets/css/partials/_hybrid-cloud-cases.scss:40]: Is this needed? `.container` already gets work done, isn't it?

**Review by trean — CHANGES_REQUESTED** [review]
Looks good. 
I have the only question: why does each section contain a wrapper with a custom class restricting its width? Seems that the `.container` class on the same element already does it.
Could you please clarify this place because I don't see the difference if I remove wrapper classes? Or fix it if it was unintentional?

File: qdrant-landing/themes/qdrant-2024/assets/css/partials/_hybrid-cloud-cases.scss (hunk 1 above)
Comment (nastyapash) [reply qdrant-landing/themes/qdrant-2024/assets/css/partials/_hybrid-cloud-cases.scss:40]: Yes, this is not necessary, I will delete it in all my PRs

Comment (trean) [reply qdrant-landing/themes/qdrant-2024/assets/css/partials/_hybrid-cloud-cases.scss:40]: Thank you so much!

**Review by trean — APPROVED** [review]

**Pull request merged**